cpp/include/cpu_operations.h

```diff
@@ -90,7 +93,13 @@ class CpuOperations {
   /// This function returns a Matrix of type T
   static Matrix<T> Multiply(const Matrix<T> &a, const Matrix<T> &b) {
     // Matrix-matrix multiplication
-    return a * b;
+    Matrix<T> result;
+    high_resolution_clock::time_point t1 = high_resolution_clock::now();
+    result = a * b;
+    high_resolution_clock::time_point t2 = high_resolution_clock::now();
+    auto duration = duration_cast<microseconds>( t2 - t1 ).count();
+    std::cout << "CPU Time: " << (long)duration << std::endl;
```

Comment:
CpuOperations should be clean of timing or anything that is not related to just computing the Multiply. You can do timing in unit tests

Reply:
Fixed.